### PR TITLE
HZC-1896: Add TLS related details for the Hazelcast Cloud Trade Monitor demo

### PR DESCRIPTION
**Pull request by sdreger**

### CODE REVIEW

File: realtime-trade-monitor/README.md [hunk 1]
```diff
@@ -67,43 +67,109 @@ Browse to localhost:9000 to see the dashboard.
 
 1. Create Enterprise Hazelcast cluster in https://cloud.hazelcast.com/
```
Comment (vladoschreiner) [inline realtime-trade-monitor/README.md]: I believe that the cluster must be set to _Public access._

File: realtime-trade-monitor/README.md [hunk 2]
```diff
@@ -67,43 +67,109 @@ Browse to localhost:9000 to see the dashboard.
...
-   by putting there cluster id and discovery token from previous step. Minimal working config example:
+
+If you're not using TLS encryption for your cluster, use the following minimal working configuration:
 ```
 hazelcast-client:
-  cluster-name: <CLUSTER_NAME>
+  cluster-name: <CLUSTER_GROUP_NAME>
   instance-name: query-client
   properties:
     hazelcast.client.cloud.url: "https://uat.hazelcast.cloud" #Optional, if env is not default
```
Comment (vladoschreiner) [inline realtime-trade-monitor/README.md]: Stay consistent with the environemnts. The first step expected the cluster to be created on PROD.

File: realtime-trade-monitor/README.md [hunk 3]
```diff
@@ -67,43 +67,109 @@ Browse to localhost:9000 to see the dashboard.
...
+5. We need to have Kafka cluster that is reachable by Hazelcast Cloud nodes. For demo purposes, the easiest way is
+   to create the simplest Kafka cluster at https://confluent.cloud with defaults.
+
+6. Create topic `trades` with 4 partitions. If you use https://confluent.cloud go to Topics section in the UI.
+
+7. Put all kafka consumer/producer properties in `trade-producer/src/main/resources/kafka.properties` and
    `trade-queries/src/main/resources/kafka.properties`. If you use https://confluent.cloud you can find them in
-   Data Integration - Client - New Client section.
+   Data Integration > Client > New Client section.
+
```
Comment (vladoschreiner) [inline realtime-trade-monitor/README.md:145]: The API_KEY and SECRET placeholders must be replaced with the values available in Confluent.cloud (Data integration > API Keys).

File: realtime-trade-monitor/README.md [hunk 4]
```diff
@@ -67,43 +67,109 @@ Browse to localhost:9000 to see the dashboard.
...
    `webapp/src/main/resources/hazelcast-client.yaml` 
-   by putting there cluster id and discovery token from previous step. Minimal working config example:
+
+If you're not using TLS encryption for your cluster, use the following minimal working configuration:
 ```
 hazelcast-client:
-  cluster-name: <CLUSTER_NAME>
+  cluster-name: <CLUSTER_GROUP_NAME>
   instance-name: query-client
   properties:
```
Comment (vladoschreiner) [inline realtime-trade-monitor/README.md:87]: Please make sure that the hazelcast client config files is valid once # character is deleted (once the cloud configs are uncommented). Especially the indentation. There are people who don't work with YAML and might get confused by how much spaces to add.

**Review by vladoschreiner — CHANGES_REQUESTED** [review]
Minor changes. The whole flow was tested and works fine.

File: realtime-trade-monitor/README.md (hunk 1 above)
Comment (sdreger) [reply realtime-trade-monitor/README.md]: Agree, added.

File: realtime-trade-monitor/README.md (hunk 4 above)
Comment (sdreger) [reply realtime-trade-monitor/README.md:87]: Both configs are valid and tested and can be safely copy-pasted, all the comments were removed.

File: realtime-trade-monitor/README.md (hunk 2 above)
Comment (sdreger) [reply realtime-trade-monitor/README.md]: Agree, removed that line, it is not needed for the `prod` env.

File: realtime-trade-monitor/README.md [hunk 5]
```diff
@@ -65,45 +65,111 @@ Browse to localhost:9000 to see the dashboard.
...
+   <artifactId>hazelcast-enterprise</artifactId>
+   <version>${hazelcast.version}</version>
+</dependency>   
+```
+
+5. We need to have Kafka cluster that is reachable by Hazelcast Cloud nodes. For demo purposes, the easiest way is
    to create the simplest Kafka cluster at https://confluent.cloud with defaults.
 
-5. Create topic `trades`. If you use https://confluent.cloud go to Topics section in the UI.
+6. Create topic `trades` with 4 partitions. If you use https://confluent.cloud go to Topics section in the UI.
```
Comment (huseyinbabal) [inline realtime-trade-monitor/README.md]: `...If you already use https://confluent.cloud, go to ...` mind the comma

File: realtime-trade-monitor/README.md [hunk 6]
```diff
@@ -120,10 +186,10 @@ java -jar trade-queries/target/trade-queries-5.0.jar ingest-trades ""
...
 
-5. Start the front end
+11. Start the front end
 
 ```
 java -jar webapp/target/webapp-5.0.jar 
 ```
 
-Browse to localhost:9000 to see the dashboard.
+12. Browse to localhost:9000 to see the dashboard.
```
Comment (huseyinbabal) [inline realtime-trade-monitor/README.md]: what about making this a real link? instead of copy paste to browser, user can just click

**Review by huseyinbabal — APPROVED** [review]
Ok with minor comments

**Review by vladoschreiner — APPROVED** [review]

**Pull request merged**